### PR TITLE
Traditional upgrades

### PR DESCRIPTION
**Pull request by C4ptainCrunch**

### CODE REVIEW

**Review by Bruno-brsy — COMMENTED** [review]
We downgrade to Python 3.10 ? Strange

**Comment by C4ptainCrunch** [comment]
It's only a comment saying from what environment the file was generated. It does not change the version of python being run on the server or the tests

**Review by Bruno-brsy — APPROVED** [review]
😲 Ho, ok, I didn't know that.

**Review by etnarek — APPROVED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -1,20 +1,20 @@
 #
-# This file is autogenerated by pip-compile with python 3.11
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.10
```
Comment (Mortinat) [inline requirements.txt:2]: The version can matter ( I don't think it will in this case) since it's on this version the tool will base the requirements

**Pull request merged**